src/java/org/apache/cassandra/service/accord/async/AsyncOperation.java

```diff
@@ -65,6 +66,7 @@
 
     private State state = State.INITIALIZED;
     private final AccordCommandStore commandStore;
+    private SafeAccordCommandStore safeStore;
```

Comment:
I don't think we need this field? Should be fine with just a local variable under `case LOADING`? Am I missing something important?